### PR TITLE
Favorites integration

### PR DESCRIPTION
**Pull request by EstebanIndiveri**
#### Summary 

Added api integration, favorite screeen, take a favorites articles by username 

some fixes interfaces and action.

#### Android 
Captura de Pantalla 2021-08-23 a la(s) 16 47 59

untitled

#### Ios
Captura de Pantalla 2021-08-23 a la(s) 16 38 09

Simulator Screen Recording - iPhone 12 - 2021-08-23 at 16 39 13

### CODE REVIEW

File: src/interfaces/reduxInterfaces.ts [hunk 1]
```diff
@@ -44,7 +44,9 @@ export interface ArticlesState {
   articlesListError: Nullable<String>;
 }
 export interface FavouritesState {
-  favouritesarticlesList: [];
+  favouritesarticlesList: {
```
Comment (john07289) [inline src/interfaces/reduxInterfaces.ts:47]: favoritesArticlesList

File: src/redux/favourites/actions.ts [hunk 2]
```diff
@@ -13,16 +11,13 @@ export const TARGETS = {
   FAVOURITES_ARTICLES_LIST: 'favouritesarticlesList'
```
Comment (john07289) [inline src/redux/favourites/actions.ts:11]: FAVORITES_ARTICLES_LIST:  'favoritesArticlesList'

File: src/services/FavouritesArticlesService.ts [hunk 3]
```diff
@@ -1,116 +1,42 @@
...
+//       bio: 'Fanático de Racing Club de Avellanada, el club más grande de Argentina. Único heptacampeón y glorioso por su gente y su historia.',
+//       image: 'https://www.elterritorio.com.ar/img/1/289/2020060820580200_1.jpg',
+//       following: false
+//     },
+//     favorited: false,
+//     favoritesCount: 1
+//   },
+// ];
+const baseURL = Config.API_BASE_URL;
+const api = create({ baseURL });
```
Comment (john07289) [inline src/services/FavouritesArticlesService.ts:26]: Can you use api from config/api?

File: src/interfaces/reduxInterfaces.ts (hunk 1 above)
Comment (EstebanIndiveri) [reply src/interfaces/reduxInterfaces.ts]: changed

File: src/redux/favourites/actions.ts (hunk 2 above)
Comment (EstebanIndiveri) [reply src/redux/favourites/actions.ts]: changed

File: src/services/FavouritesArticlesService.ts (hunk 3 above)
Comment (EstebanIndiveri) [reply src/services/FavouritesArticlesService.ts]: I tried again and it's work idk why, change import to '@config/api';

**Review by andyOlarte514 — APPROVED** [review]

**Pull request merged**